app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
```

Comment:
Consider adding a descriptive error message to your `OutdatedVaccineError` class. For example, `class OutdatedVaccineError(VaccineError):
    def __init__(self, message="The vaccine is outdated."):
        super().__init__(message)`